streampipes-client-python/streampipes_client/model/common.py

```diff
@@ -55,30 +55,33 @@ class BaseElement(BasicModel):
     element_id: Optional[StrictStr]
 
 
-class EventPropertyQualityRequirement(BaseElement):
+class EventPropertyQualityRequirement(BasicModel):
     """
     Data model of an `EventPropertyQualityRequirement` in compliance to the StreamPipes Backend.
     """
 
+    element_id: Optional[StrictStr]
     minimum_property_quality: Optional[BaseElement] = Field(alias="eventPropertyQualityDefinition")
     maximum_property_quality: Optional[BaseElement] = Field(alias="eventPropertyQualityDefinition")
```

Comment:
The EventPropertyQualities and EventPropertyQualityRequirement aren't used by the API anymore and I think we could remove them. The elementId is also not used for every class anymore and we probably should change this. But I can do this in another PR too.

Reply:
That would be great 👍